bin/control.sh

```diff
@@ -61,7 +61,11 @@ case ${COMPONENT} in
                 FLYWAY_EXECUTABLE="$FLYWAY_DIR/flyway"
                 SCRIPTS_LOCATION="$FLYWAY_DIR/$DATABASE_SCRIPT_DIR"
 
-                env FLYWAY_LOCATIONS="filesystem:$SCRIPTS_LOCATION" "$FLYWAY_EXECUTABLE" migrate -url="$DB_URL" -user="$DB_USERNAME" -password="$DB_PASSWORD" || exit 1
+                # To hide the db password, the flyway command is displayed with all arguments except password and then executed with password in hidden mode
+                FLYWAY_COMMAND="filesystem:$SCRIPTS_LOCATION $FLYWAY_EXECUTABLE migrate -url=$DB_URL -user=$DB_USERNAME -password="
+                set +x
+                env FLYWAY_LOCATIONS=${FLYWAY_COMMAND}$DB_PASSWORD || exit 1
```

Comment:
You can put `FLYWAY_LOCATIONS=` into the above command on line 65, no reason to set that here

Reply:
Where do you echo out the command for debugging?

Reply:
Oh it's done because of set -x

Reply:
Line 65 - to store the command without password; as set -x is there, the command is expanded and echoed
Line 67- where the command is executed with password; it is hidden

Reply:
@Niveditha-phData how about this?
> You can put FLYWAY_LOCATIONS= into the above command on line 65, no reason to set that here